<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QDialog" name="Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>80</y>
     <width>741</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_CPU">
    <widget class="QTableWidget" name="tableWidget_CPU">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>621</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_storage">
    <widget class="QTableWidget" name="tableWidget_storage">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>621</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_storage">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>370</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_memory">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Used memory:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_storage">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>370</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_CPU">
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_storage">
      <property name="text">
       <string>Internal Storage</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
